sdk/search/Azure.Search.Documents/src/SearchServiceClient.cs

```diff
@@ -193,27 +202,20 @@ public virtual SearchIndexClient GetSearchIndexClient(string indexName)
         /// This operation returns the number and type of objects in your
         /// service, the maximum allowed for each object type given the service
         /// tier, actual and maximum storage, and other limits that vary by
-        /// tier.  This request pulls information from the service so that you
+        /// tier. This request pulls information from the service so that you
```

Comment:
I consider doc comments to be a whitespace sensitive language and this is a syntax error.

_(+@jsquire and @annelo-msft  to back me up here...)_

Reply:
I know emacs isn't a far cry from a typewriter 😉, but the XML doc to HTML and/or markdown renderer canonicalizes it *properly* anyway.

Reply:
_Ohhh..._  I didn't realize that Vim uses a variable width font.  That's cute.  Two spaces would render the same in our ref docs, but I thought you were using a real editor where the additional space was a useful affordance.  😄